css/index.css

```diff
@@ -172,4 +173,127 @@ header .logo {
 footer {
     text-align: center;
     margin: 40px 0 20px;
+}
+
+/* ADAPTIVE STYLES BELOW: */
+
+
+/* Tablet */
+@media (max-width: 768px) {
+
+    .container {
+        width: 100%;
+        max-width: 768px;
+    }
+
+    header nav {
+        width: 59.5%; /* 457px / 768px = 59.5% */
+        margin-left: 2.6%; /* 20px / 768px = 2.6% */
+    }
+
+    header .logo {
+        margin: 28px 0 0 7.68%; /* 59px / 768px = 7.68% */
+    }
+
+    .cta {
+        justify-content: center;
+    }
+    
+    .cta .cta-text {
+        margin: 0 5.2% 0 0; /* 40px / 768px = 5.2% */
+    }
+
+    .main-content .middle-img {
+       width: 100%;
+    }
+
+    .main-content {
+       margin: 75px 2.6% 35px;  /* 20px / 768px = 2.6% */
+    }
+}
+
+/* Phone */
+@media (max-width: 400px) {
+    
+    .container {
+        width: 100%;
+    }
+
+    header {
+        flex-wrap: wrap;
+        justify-content: center;
+    }
+
+    header nav {
+        width: 100%;
+    }
+
+    header nav {
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        margin: 0;
+        align-items: center;
+    }
+
+    header nav {
+        width: 100%;
+        margin: 0;
+    }
+
```

Comment:
You have three different selectors for the `header nav`.  You can delete the first and the last ones but keep 231 through 237